@types/index.test-d.ts

```diff
@@ -95,8 +88,7 @@ async function run() {
 		['b', '2'],
 		new Map([['a', null], ['3', null]]).keys()
 	]);
-
-	fetch.isRedirect = (code: number) => true;
```

Comment:
This was removed since this is no longer possible. `isRedirect` is now exported with `export function isRedirect` and is thus no longer a property on the `fetch` function...